src/stories/components/Sidebar/styles.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Sidebar
+
+.sidebar {
```

Comment:
There are no styles apply to this. The layout should be dictated by the `sidebar` not the child content; remove the styles you have and replace them with these:

```
sidebar {
    @include bp-tablet-lg {
        display: flex;
    }
}

.sidebar__aside {
    flex-basis: rem(210px);

    @include bp-tablet-lg {
        margin-right: $spacing-3;
    }

    @include bp-desktop {
        margin-right: $spacing-5;
    }
}

.sidebar--right {
    .sidebar__aside {
        ...
        }
    }
}
```

Note that this is just to get you started. You'll have to adjust the display and styles for the `sidebar--right`.

Reply:
Awesome, thank you!